extension/js/common/composer.ts

```diff
@@ -84,6 +84,8 @@ export class Composer {
   private composeWindowIsMaximized = false;
   private refBodyHeight?: number;
   private urlParams: ComposerUrlParams;
+  private keyImportUI = new KeyImportUi({});
+  private removeValidationElements: (() => void) | undefined;
```

Comment:
should be called `rmPwdValidationElements` - because otherwise it's hard to guess what kind of validation this refers to